templates/partners/iot-device.html

```diff
@@ -0,0 +1,123 @@
+{% extends 'base_index.html' %}
+
+{% block meta_description %}For nearly two decades, Canonical and our enterprise partners have provided market-leading solutions for customers seeking alternatives to complex and costly proprietary operating environments.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1qNEjihmySR6l6LZJ-SkSVTpaBSxuY8imoseN0OLkiMw/edit#{% endblock %}
+
+{% block title %}IoT Device Programme | Partners{% endblock %}
+
+{% block meta_image %}https://assets.ubuntu.com/v1/a491f87c-Ubuntu+Certified+Hardware.svg{% endblock %}
+
+{% block content %}
+<section class="p-strip--suru-bottomed is-deep" style="padding-bottom: 8rem;">
```

Comment:
Does this need to be deep? The padding bottom adds a lot of space on small screens.

Reply:
I think it needs to be deep because it looks like this.
![image](https://user-images.githubusercontent.com/57550290/144206181-44e57c58-8304-4437-9ed7-d33066472882.png)
I removed the `padding-bottom` style instead.